spec/checker_spec.lua

```diff
@@ -130,14 +130,6 @@ describe("Pallene type checker", function()
             "duplicate field 'x' in table")
     end)
 
-    it("prohibits tables with fields with more than LUAI_MAXSHORTLEN chars", function()
-        local field = string.rep('a', 41)
-        assert_error([[
-            function f(t: {]].. field ..[[: float}) end
-        ]],
-            "field name '".. field .. "' too long")
-    end)
-
```

Comment:
This test case should be moved to coder_spec. Except that instead of verifying that it produced an error, it should verify that it is working correctly.